net/modemmanager/files/modemmanager.proto

```diff
@@ -313,13 +313,15 @@ proto_modemmanager_init_config() {
 	proto_config_add_string	 pincode
 	proto_config_add_string	 iptype
 	proto_config_add_boolean lowpower
+	proto_config_add_int timeout
 	proto_config_add_defaults
 }
 
 proto_modemmanager_setup() {
 	local interface="$1"
 
-	local modempath modemstatus bearercount bearerpath connectargs bearerstatus beareriface
+	local modempath modemstatus bearercount bearerpath connectargs bearerstatus beareriface timeout
+	[ "$timeout" = "" ] && timeout="10"
```

Comment:
I assume you want the "timeout" setting to be part of the modem settings in `/etc/config/network`? if so, this check to intiialize timeout to 10 if not already set should probably be done after having run `json_get_vars` below.